tests/integrations/train/test_runners.py

```diff
@@ -47,6 +48,7 @@ def _run_and_check_output_files(mocker, tmp_path, config):
     mock_flags = mocker.patch("vmcnet.train.runners.FLAGS")
     mock_flags.config = config
     mock_flags.reload = train.default_config.get_default_reload_config()
+    mock_flags.preset_config = ConfigDict({"path": "NONE"})
```

Comment:
Nir: prefer to define a constant like `NO_PRESET_CONFIG="NONE"` i.e. in parse_config_flags and use that in all three places where you have "NONE", just general good practice to avoid accidental mismatching strings

Reply:
updated